src/cmd/turnkey/pkg/wallets.go

```diff
@@ -19,11 +20,18 @@ var (
 )
 
 func init() {
-	walletCreateCmd.Flags().StringVar(&walletNameOrID, "name", "", "name to be applied to the wallet")
+	walletCreateCmd.Flags().StringVar(&walletNameOrID, "name", "", "name to be applied to the wallet.")
+
+	walletInitImportCmd.Flags().StringVar(&user, "user", "", "ID of user to importing the wallet")
```

Comment:
For things like this in a common namespace, it would be safer to additionally namespaces the package-scoped variable here.  Perhaps something like user -> walletInitImportUser?